### PR TITLE
Add updated_at to hex_pb_names

### PR DESCRIPTION
**Pull request by wojtekmach**

### CODE REVIEW

**Comment by wojtekmach** [comment]
@ericmj we recently discussed this but just double-checking, do we still want to add the checksum (per https://github.com/hexpm/hexpm/issues/664) or `updated_at` is enough for now? I think it is enough as it will keeping incrementing even when we publish non-latest releases, so it should be a good indicator when things changed (so that mirror would re-sync, for example.)

File: proto/hex_pb_names.proto [hunk 1]
```diff
@@ -13,4 +13,7 @@ message Package {
 
   // If set, the name of the package repository (NEVER USED, DEPRECATED)
   // string repository = 2;
+
+  // unix timestamp
+  optional fixed32 updated_at = 3;
```
Comment (ericmj) [inline proto/hex_pb_names.proto:18]: Can we use this type https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp or at least something more accurate with sub-second resolution?

**Comment by ericmj** [comment]
> we recently discussed this but just double-checking, do we still want to add the checksum (per hexpm/hexpm#664) or updated_at is enough for now? I think it is enough as it will keeping incrementing even when we publish non-latest releases, so it should be a good indicator when things changed (so that mirror would re-sync, for example

Let's add only updated_at for now and close that issue. If there is still a need for checksums after this a new issue can be opened.

File: proto/hex_pb_names.proto (hunk 1 above)
Comment (wojtekmach) [reply proto/hex_pb_names.proto]: Sounds good, see: https://github.com/hexpm/hex_core/pull/114/files#diff-61531a44d8f89a5934e78d85759a628de1a951d06c03b74ccbc7036dada837a0R22

**Review by ericmj — APPROVED** [review]

**Pull request merged**